pandas/core/arrays/datetimes.py

```diff
@@ -998,48 +998,62 @@ def date(self):
         'dim',
         "The number of days in the month")
     daysinmonth = days_in_month
-    is_month_start = _field_accessor(
-        'is_month_start',
-        'is_month_start',
-        "Logical indicating if first day of month (defined by frequency)")
-    is_month_end = _field_accessor(
-        'is_month_end',
-        'is_month_end',
-        """
-        Indicator for whether the date is the last day of the month.
+    _is_month_doc = """
+        Indicates whether the date is the {first_or_last} day of the month.
 
         Returns
         -------
         Series or array
-            For Series, returns a Series with boolean values. For
-            DatetimeIndex, returns a boolean array.
+            For Series, returns a Series with bool.
+            For DatetimeIndex, returns a bool array.
 
         See Also
         --------
-        is_month_start : Indicator for whether the date is the first day
-            of the month.
+        is_month_{end_or_start} : Return a boolean indicating whether the date
+            is the {last_or_first} day of the month.
```

Comment:
in the See Also you can also have both methods, so we don't need so many variables.